### ISSUE TITLE
weather missing for May 24, 2008

### ISSUE THREAD

**Issue by dpesce** [Smithsonian/ngehtsim#31]
For some reason, the weather information for this single date (May 24, 2008) seems to be missing.

**Comment by wumpus** [comment]
If it were NOMADS the help desk is [ncep.pmb.dataflow@noaa.gov](mailto:ncep.pmb.dataflow@noaa.gov)

But it's probably MERRA-2 that you're asking about. I can't find an email address for them

I'd definitely try to download that date again before asking a human.

**Comment by dpesce** [comment]
Yeah I think it's most likely some issue with the compression I run after doing the radiative transfer, rather than something with MERRA-2 itself.  I think I'll re-download the MERRA-2 data and then re-run the tabulation, and then if that still shows issues then I'll consider trying to find some contact info for someone.

**Comment by wumpus** [comment]
The nice thing about NOMADS is that you can download a box of data from a particular forecast run for a particular time in that run... very granular. That's how I can cough up new plots just 30 minutes after the forecast finishes.
